tests/conftest.py

```diff
@@ -3,16 +3,16 @@
 
 
 @pytest.fixture()
-def RubyGem(Command):
+def RubyGem(host):
```

Comment:
We should be able to remove this entire file in favor of the rubygem resource built into Testinfra in recent releases.

Reply:
I'm having a hard time finding any documentation on that. Can you point me in the right direction?

Reply:
I was mistaken. There had been some back and forth a while back and I thought that a gem module had been added. We can take a look at upstreaming this later.